vercel.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rewrites": [{ "source": "/(.*)", "destination": "/" }]
+}
```

Comment:
SPA-specific rewrite rule.
Similar to the previous nginx config file `spa.conf`.